lib/launcher.rb

```diff
@@ -189,19 +269,19 @@ def launch
       exit(1)
     end
 
-    def install_webrick_callback!(*apps)
+    def install_http_server_callback!(*apps)
       apps.compact!
 
-      # track how many webrick apps are still starting up
-      @pending_webrick = apps.size
-      @pending_webrick_lock = Mutex.new
+      # track how many apps are still starting up
+      @pending_server = apps.size
+      @pending_server_lock = Mutex.new
 
       apps.each do |app|
         # add a callback to each server, decrementing the pending counter
         app[:StartCallback] = lambda do
```

Comment:
This callback doesn't seem to be called anymore, which breaks systemd startup notification.

Reply:
Thanks for the insight, I've fixed this and I am currently working on unit tests.